src/cryptsetup/cryptsetup-generator.c

```diff
@@ -670,6 +671,7 @@ static int run(const char *dest, const char *dest_early, const char *dest_late)
         assert_se(arg_dest = dest);
 
         arg_crypttab = secure_getenv("SYSTEMD_CRYPTTAB") ?: "/etc/crypttab";
+        arg_run_systemd_cryptsetup = secure_getenv("SYSTEMD_RUN_SYSTEMD_CRYPTSETUP") ?: "/run/systemd/cryptsetup";
```

Comment:
hmm, I think it would be nicer to use `$RUNTIME_DIRECTORY` for this, i.e. the variable `RuntimeDirectory=` sets too. We are talking about the same concept here, but `$RUNTIME_DIRECTORY` is the generic name for it, which is much preferable I think. An no, we wouldn't actually set RuntimeDirectory= in any unit file (we are talking about a generator here after all) but I still think we should use the same names for the same concepts.